src/patternfly/components/TreeView/tree-view.scss

```diff
@@ -110,12 +110,15 @@ $pf-c-tree-view--MaxNesting: 10;
 }
 
 .pf-c-tree-view__node-toggle {
+  position: absolute;
+  left: calc(var(--pf-c-tree-view__node--PaddingLeft) - var(--pf-global--spacer--lg) - var(--pf-global--spacer--lg));
```

Comment:
Normally I would add a `top` value here, but it seems to be working without it. Maybe someone can explain why that works some day? If we wanted to add that, it should work with `top: var(--pf-c-tree-view__node--PaddingTop)`